specifyweb/frontend/js_src/lib/components/AppResources/Create.tsx

```diff
@@ -258,8 +265,7 @@ function EditAppResource({
     () =>
       deserializeResource(
         addMissingFields(type.tableName as 'SpAppResource', {
-          // I don't think this field is used anywhere
-          level: 0,
+          level: scopeLevel[directory.scope as keyof typeof scopeLevel],
```

Comment:
does this place need to be refactored too?
https://github.com/specify/specify7/blob/issue-4170/specifyweb/frontend/js_src/lib/components/AppResources/EditorWrapper.tsx#L62C11-L62C17

Reply:
Ben and I were pretty sure this field wasn't used by sp6 and sp7 anywhere. Can you show what it is used by now?

Reply:
it's used to organized the views by scope

Reply:
Can you please link to where in the code it is used???

Reply:
https://github.com/specify/specify7/blob/7caa59814a7a81564aff541f1be0a29cc95aeecb/specifyweb/frontend/js_src/lib/components/DataModel/scoping.ts#L75

I am actually not 100% sure, I found the solution when using dev tools and seeing that level was always assigned to the same number but determined where it was stored in the 'menu tree'

Reply:
I don't see any references to the `level` variable in either of these two files. Your code change is setting the `level` variable. Unless something is looking at the value of that variable, this change is needless.

Can you link to the exact line of code where the value of the `level` variable is read?
If not, can you double check that the bug does occur as soon as you revert this change, and is fixed as soon as you re-apply the change?

Reply:
FYI: The location of the app resource in the app resource tree is determined solely by which spAppResourceDir the app resource belongs too. `level` field was added long time ago by sp6, but it's very redundant and so was never used AFAIK.
So when debugging the problem, look at what spAppResourceDir is attached to this app resource, and if it is what you expect

Reply:
there is an issue with global level which I understood wrong. My fixes in this branch bring nothing. I will revert and try from scratch to find the problem...